packages/core/src/routes/session.ts

```diff
@@ -28,28 +33,20 @@ import { AnonymousRouter } from './types';
 
 export default function sessionRoutes<T extends AnonymousRouter>(router: T, provider: Provider) {
   router.post(
-    '/session',
+    '/session/sign-in/:signInFlowType',
```

Comment:
can be seperated into specific routes.
`/session/sign-in/username-and-password`
`/session/sign-in/phone`
to avoid if..else if.. else if .. else